### PR TITLE
Fixes header color and message

### PR DESCRIPTION
**Pull request by drzackyll**
[ch15238]

### CODE REVIEW

File: src_ts/components/app-shell/header/page-header.ts [hunk 1]
```diff
@@ -99,13 +99,17 @@ export class PageHeader extends connect(store)(LitElement) {
...
+  environment: string = 'LOCAL';
+
   @property({type: Array})
   editableFields: string[] = ['office', 'section', 'job_title', 'phone_number', 'oic', 'supervisor'];
 
   public connectedCallback() {
     super.connectedCallback();
     this.setBgColor();
-    this.isStaging = isStagingServer();
+    this.isProduction = isProductionServer();
```
Comment (adi130987) [inline src_ts/components/app-shell/header/page-header.ts]: move this line inside `_checkEnvironment` method

File: src_ts/components/app-shell/header/page-header.ts [hunk 2]
```diff
@@ -156,6 +160,13 @@ export class PageHeader extends connect(store)(LitElement) {
     return modifiedFields;
   }
 
+  protected _checkEnvironment() {
```
Comment (adi130987) [inline src_ts/components/app-shell/header/page-header.ts]: remove `_` from method name

File: src_ts/components/app-shell/app-theme.ts [hunk 3]
```diff
@@ -14,8 +14,8 @@ documentContainer.innerHTML = `
         --secondary-text-color: rgba(0, 0, 0, 0.54);
 
         --header-color: #ffffff;
-        --header-bg-color: var(--primary-color);
-        --nonprod-header-color: #233944;
+        --header-bg-color: #233944;;
+        --nonprod-header-color: #a94442;
```
Comment (adi130987) [inline src_ts/components/app-shell/app-theme.ts]: very ugly color :)

Comment (drzackyll) [reply src_ts/components/app-shell/app-theme.ts]: Agreed!

**Pull request merged**